.github/workflows/build.yml

```diff
@@ -352,6 +352,7 @@ jobs:
         with:
           token: ${{ env.PHOTON_PAT }}
           repository: icosa-mirror/photon-fusion
+          ref: v1.1.10
```

Comment:
Without this, it will take the latest. We probably should pin it, though, so this is a good change.